src/domains/_constants/frameWorkPresets.ts

```diff
@@ -0,0 +1,56 @@
+export const FrameWorkPresets: {
+  [key: string]: { buildCommand: string; buildDirectory: string };
```

Comment:
혹시 Framework 타입의 값들을 string 대신 사용할 수 있지 않았을까요?
```suggestion
  Record<Framework, { buildCommand: string; buildDirectory: string }>;
```

Reply:
오오.. 👍👍👍👍👍